### ISSUE TITLE
Drawer/List acts different on DEV/PROD

### ISSUE THREAD

**Issue by marxus85** [material-components/material-components-web-react#562]
on DEV: 
(using  npm run start)
highlights the first item as expected, also keyboard navigation works, 
when selecting a different item list, works as expected
![dev](https://user-images.githubusercontent.com/820742/50418541-90130d00-0834-11e9-82e1-a76cff4d9635.PNG)

on PROD:
(using npm run build)
![build](https://user-images.githubusercontent.com/820742/50418561-b6d14380-0834-11e9-9c89-2c9f757d3116.PNG)
no highlighting anything, also keyboard navigation does not works, not selecting an item list

`  "dependencies": {
    "@material/react-button": "^0.7.1",
    "@material/react-drawer": "^0.7.1",
    "@material/react-list": "^0.7.1",
    "@material/react-material-icon": "^0.7.1",
    "@material/react-top-app-bar": "^0.7.1",
    "node-sass": "^4.11.0",
    "react": "^16.7.0-alpha.2",
    "react-dom": "^16.7.0-alpha.2",
    "react-scripts": "2.1.1"
  },`

**Comment by moog16** [comment]
are you using SSR? This issue is similar to yours, and it might be the cause https://github.com/material-components/material-components-web-react/issues/492

**Comment by moog16** [comment]
Is this still an issue for you?

**Comment by moog16** [comment]
Closing this is a stale issue.
